src/wp-includes/class-wp-query.php

```diff
@@ -3533,7 +3535,7 @@ public function get_posts() {
 
 			if ( $q['cache_results'] ) {
 				if ( $is_unfiltered_query && $unfiltered_posts === $this->posts ) {
-					update_post_caches( $this->posts, $post_type, $q['update_post_term_cache'], $q['update_post_meta_cache'] );
+					update_post_caches( $this->posts, $post_type_prime, $q['update_post_term_cache'], $q['update_post_meta_cache'] );
```

Comment:
This will cause any previous cases that would have passed a null value for $post_type (which would evaluate as `post`) to now be evaluated as `any`. This is probably fine, but want to ensure this is an intentional change.

Reply:
@joemcgill Before, any empty value would have been converted to 'any' in update_post_caches. 

See 
https://github.com/WordPress/wordpress-develop/blob/fe1df4023c8344dd4bb21035563e58ddd1731949/src/wp-includes/post.php#L7331-L7333

Reply:
For most empty values that's true. But `null` would have been converted to the default param value of 'post'. Again, probably fine, just wanted to make that observation clear.